src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -5639,7 +5639,7 @@ void M()
         }
     }
 }",
-                Exceptions($"\r\n{WorkspacesResources.Exceptions_colon}\r\n  MyException1\r\n  MyException2\r\n  int\r\n  double\r\n  Not_A_Class_But_Still_Displayed"));
+                Exceptions($"\r\n{WorkspacesResources.Exceptions_colon}\r\n  MyException1:\r\n  MyException2:\r\n  int:\r\n  double:\r\n  Not_A_Class_But_Still_Displayed:"));
```

Comment:
These changes seem undesirable

Reply:
The Conversion to TaggedText on the types when it finds the text has nothing in it seems to add the colons anyway.